static/app/components/events/eventTagsAndScreenshot/screenshot/index.tsx

```diff
@@ -118,22 +115,20 @@ function Screenshot({event, attachments, organization, projectSlug, onDelete}: P
   }
 
   return (
-    <Role role={organization.attachmentsRole}>
+    <Role organization={organization} role={organization.attachmentsRole}>
```

Comment:
the `Role` component is already wrapped with `withOrganization` but passing the org here was needed for the test to work. It was the simplest solution